Project.toml

```diff
@@ -19,7 +20,7 @@ SparseDiffTools = "47a9eef4-7e08-11e9-0b38-333d64bd3804"
 TimerOutputs = "a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f"
 
 [compat]
-CUDA = "~2.3, 3.0"
+CUDA = "~2.3, ~3.2"
```

Comment:
Sorry, this creates a conflict with my push to develop where I forced `2.6` instead of `2.3`. `2.3` has issues on Summit.

Reply:
This is resolved